src/components/views/elements/Field.js

```diff
@@ -135,6 +138,7 @@ export default class Field extends React.PureComponent {
 
     render() {
         const { element, prefix, onValidate, children, ...inputProps } = this.props;
+        delete inputProps.tooltip; // needs to be removed from props but we don't need it here
```

Comment:
`delete` forces the JIT to deoptimise the object, but probably doesn't matter for this case...

Reply:
interesting - I started with a nop `!tooltip` although it sort of seems awful.